modules/cr_rich_text_paragraph/cr_rich_text_paragraph.info.yml

```diff
@@ -20,8 +20,8 @@ config_devel:
   - field.field.paragraph.cr_rich_text_paragraph.field_bg_single_msg_layout
   - field.field.paragraph.cr_rich_text_paragraph.field_background
   - field.field.paragraph.cr_rich_text_paragraph.field_body
-  - image.style.large_1700x500
-  - image.style.large_1700x500_2x
+  - image.style.large_1700x450
```

Comment:
You shouldn't delete the old ones, that should be part of the remove old config ticket